src/ORM/Rule/ExistsIn.php

```diff
@@ -40,14 +40,24 @@ class ExistsIn
     protected $_repository;
 
     /**
+     * Setup Options
+     *
+     * @var array
+     */
+    protected $_setupOptions;
+
+    /**
      * Constructor.
      *
      * @param string|array $fields The field or fields to check existence as primary key.
      * @param object|string $repository The repository where the field will be looked for,
      * or the association name for the repository.
      */
-    public function __construct($fields, $repository)
+    public function __construct($fields, $repository, $setupOptions = array())
     {
+        $this->_setupOptions = [
+                'allowPartialSchemaNulls' => true,
+            ] + $setupOptions;
```

Comment:
I think this needs to be be `$options + $defaults`. Right now, it is impossible to make `allowPartialSchemaNulls = false`.